content/rs/administering/designing-production/networking/port-configurations.md

```diff
@@ -5,69 +5,58 @@ weight: $weight
-| TCP | 8001 | Internal, External | Traffic from application to RS [Discovery Service]({{< relref "/rs/concepts/data-access/discovery-service.md" >}}) |
-| TCP | 8002, 8004, 8006 | Internal | System health monitoring |
+| TCP | 8001 | Internal, External | Traffic from application to Redis Enterprise Software [Discovery Service]({{< relref "/rs/concepts/data-access/discovery-service.md" >}}) |
+| TCP | 8070, 8071 | Internal, External | Metrics exported and managed by the web proxy |
 | TCP | 8443 | Internal, External | Secure (HTTPS) access to the management web UI |
-| TCP | 8444, 9080 | Internal | For web proxy <-> cnm_http/cm traffic |
-| TCP | 9081 | Internal, Active-Active | For Active-Active management |
-| TCP | 8070, 8071 | Internal, External | For metrics exported and managed by the web proxy |
+| TCP | 9081 | Internal, Active-Active | Active-Active management |
 | TCP | 9443 (Recommended), [8080](#turning-off-http-support) | Internal, External, Active-Active | REST API traffic, including cluster management and node bootstrap |
 | TCP | 10000-19999 | Internal, External, Active-Active | Database traffic |
-| TCP | 20000-29999 | Internal | Database shard traffic |
 | UDP | 53, 5353 | Internal, External | DNS/mDNS traffic |
+| ICMP | * | Internal | Connectivity checking between nodes |
+| TCP | 1968 | Internal | Proxy traffic |
+| TCP | 3333-3341, 3343-3344, 36379, 36380 | Internal | Internode communication |
+| TCP | 20000-29999 | Internal | Database shard traffic |
+| TCP | 8002, 8004, 8006 | Internal | System health monitoring |
+| TCP | 8444, 9080 | Internal | Traffic between web proxy and cnm_http/cm |
 
-Connection sources are:
-
-- Internal - The traffic is from other cluster nodes
-- External - The traffic is from client applications or external monitoring resources
-- Active-Active - The traffic is from clusters that host Active-Active databases
-
-## Changing the management web UI port
-
-If for any reason you want to use a custom port for the RS Web UI
-instead of the default port (8443), you can change the port. Before you
-change the RS Web UI port, make sure that the new port is not in
-use by another process.
-
-{{< note >}}
-After you change the RS Web UI port, when you add a new node to the
-cluster you must connect to the web UI with the custom port number:
+## Change the admin console port
 
-`https://newnode.mycluster.example.com:`**`<nonstandard-port-number>`**
-{{% /note %}}
+The Redis Enterprise Software admin console uses port 8443, by default. You can change this to a custom port as long as the new port is not in use by another process.
 
-To change the default port for the RS Web UI, on any node in the cluster run:
+To change this port, run:
 
 ```sh
 rladmin cluster config cm_port <new-port>
 ```
 
-## Disabling HTTP support for API endpoints
+After changing the Redis Enterprise Software web UI port, you must connect any new node added to the cluster to the UI with the custom port number:
```

Comment:
```suggestion
New nodes added the cluster will not default to the new admin console port number. You must manually connect the new nodes to the admin console: 
```

How about we reword to this? @banker @AlonMagrafta

Reply:
@kaitlynmichael This is the sentence that I don't understanding the meaning of. What does it mean to manually connect new nodes to the admin console?